Automattic/woocommerce-payments
# Remove temporary transaction table search field CSS that was used for Payment Activity Card report filtering

Fixes #9736

#### Changes proposed in this Pull Request

#8996 has introduced a temporary CSS fix used to improve the transaction table search field UX when many search terms were active. This occurred when clicking through from a Payment Activity Card report (currently unreleased).

This also inadvertently impacted the Payouts list styles (issue #9736) by misaligning the "Download" button to the left of the table header, inconsistent with other list view tables (Transactions, Disputes).

PR #9871 negates the need for this CSS by using a method other than search bar terms for filtering Payment Activity Card reports.

This PR removes these CSS, resolving the Payouts list "Download" button styling issue and removing dead code that no longer needs to be maintained.

Payouts:
![payouts](https://github.com/user-attachments/assets/c9da57d2-295e-46ad-947c-8663ceb302f8)

Transactions:
![transactions](https://github.com/user-attachments/assets/bbc767fd-1e07-4960-9342-daf2f7f5d6be)

#### Testing instructions





* View the Transactions List view, adding a search filter, e.g. an email address, and confirm that the styling of this table remains correct, including on mobile viewports
* Ensure that the Download button on the Payouts, Disputes and Transactions tables are aligned consistently

-------------------

- [x] Covered with tests – **Note** this is a styling issue, so manual testing is suitable as we don't have a visual regression testing process in place.

**Post merge**
